### DISCUSSION TITLE
Documenation for using a trained model for inference?

### DISCUSSION THREAD

**Discussion by jpmorris** [explosion/spaCy#13574] [Help: Coding & Implementations] [answered]
I've looked high and low and I don't see any example for how to actually use a trained model to score on unseened data--say in a production pipeline with new data coming in at a regular cadence.  I did find a stackoverflow post  which said I need to load the train model and create an  nlp object.  I assume since the nlp object is just a pipline that uses existing trained models to score examples, I can just use `nlp.pipe` to run over a DocBin of new data? But is there really not a single example of how to do this in any of the documentation?  Can I use the CLI for this or do I have to make a script to do it?

**Comment by honnibal** [comment, marked as answer]
Does this command do what you need? https://spacy.io/api/cli#apply

**Comment by jpmorris** [comment]
That looks good to me! Thanks for the response and quick reply!
